userspace/libsinsp/cri.proto

```diff
@@ -727,7 +727,7 @@ message ContainerConfig {
     // use-cases (e.g. debugging).
     // TODO: Determine if we need to continue supporting these fields that are
     // part of Kubernetes's Container Spec.
-    bool stdin = 12;
+    bool f_stdin = 12;
```

Comment:
I remember that when we did the musl work this was needed for reserved keywords under musl. However, I also remember that we didn't just rename directly to avoid synchronization issues in the future.

I think that a better option here would be to have a script that does the job of downloading and applying our changes for contributors to use to generate this file.

I know that proto was made for bc but I made this file here https://gist.github.com/fntlnz/149ac1bbca57061d896ba6f3361c3361 and I feel like a script that keeps things in sync would be beneficial now and in the future for a better understanding of what the API exposes, where the source of the file is (now we don't even link it, right) and things like that.

Reply:
Will give it a try, right now I have just pushed a rebased version.